upcloud/error_codes.go

```diff
@@ -0,0 +1,211 @@
+package upcloud
+
+// Error codes used in upcloud.Error struct
+const (
+	ErrCodeAuthenticationFailed                           = "AUTHENTICATION_FAILED"
```

Comment:
We could type these untyped consts as `string`

Reply:
Good point